modules/rostests/apitests/crt/__dtoi64.c

```diff
@@ -0,0 +1,114 @@
+/*
+ * PROJECT:         ReactOS API tests
+ * LICENSE:         GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:         Tests for __dtoi64
+ * COPYRIGHT:       Copyright 2021 Roman Masanin <36927roma@gmail.com>
+ */
+
+#include <apitest.h>
+
+#define WIN32_NO_STATUS
+#include <stdio.h>
+#include <stdlib.h>
+#include <errno.h>
+
+START_TEST(__dtoi64)
+{
+    double dValue;
+    long long iValue;
+    long long* pDValue = (long long*)&dValue;
+
+    /* test 32bit number */
```

Comment:
Because all these tests are somehow similar, I wonder whether it could be possible to place them in an array, where each entry is: `{ double_value_to_test , expected_long_long_value }, /* test something */`
and then you do a loop:
```c
unsigned int i;
for (i = 0; i < _countof(test_array); ++i)
{
    dValue = test_array[i].double_value;
    iValue = (long long)dValue;
    ok(iValue == test_array[i].expected_longlong, "Expected %lld, but %lf -> %lld\n", test_array[i].expected_longlong, dValue, iValue);
}
```